### DISCUSSION TITLE
native messenger opens edits in "notepad" instead of gvim

### DISCUSSION THREAD

**Discussion by imadrafi** [tridactyl/tridactyl#4372] [Q&A] [answered]
Hi guys, 
Using Firefox on win11.

When I hit ctrl+i on text box to make an edit tridactyl opens notepad (instead of vim). 
Unable to work out how to configure tridactyl/firefox native messenger to open edits in vim/gvim instead of notepad

Any help? Thanks

**Comment by imadrafi** [comment, marked as answer]
Update:
My issue is resolved. 
I set my editor in tridactyl using editorcmd. I used:

:set editorcmd start cmd.exe /c vim.exe

This worked for me. Ctrl+i now opens up my edits in vim. 

I did have to add vim.exe to %PATH% variable. 


Thanks user sotto (sotto) on Element who pointed me to the editorcmd!

**Comment by sandrock** [comment]
I used:

```
set editorcmd "C:\Program Files (x86)\Vim\vim82\gvim.exe"
```
